src/gridded/gridded.jl

```diff
@@ -24,11 +24,11 @@ function GriddedInterpolation{N,TCoefs,TWeights<:Real,IT<:DimSpec{Gridded},pad}(
         issorted(k) || error("knot-vectors must be sorted in increasing order")
         iextract(IT, d) != NoInterp || k == collect(1:size(A, d)) || error("knot-vector should be the range 1:$(size(A,d)) for the method Gridded{NoInterp}")
     end
-    c = one(TWeights)
+    c = zero(TWeights)
```

Comment:
At one point we couldn't trust that `zero(T)*zero(T)` had type `T`: for example, I believe `0x00*0x00` was a `UInt64`. It's not really relevant for most types anymore. If one attached units to the weights, then the multiplications might still make sense, but though we can (or should be able to---I have haven't actually tested) interpolate unitful quantities, I don't see a reason that the weights should be unitful.